src/app/event-card/event-card.component.html

```diff
@@ -3,10 +3,11 @@
     <button
       mat-raised-button
       color="primary"
-      (click)="addTask()"
+      style="background-color: green;"
```

Comment:
why did you remove the previous functilnaity? this button is used by other orgs to add trips themselves - I think it should be a different button.

Also - it is restricted with the `isDsptacher` if, a driver is not a dispatcher so a common driver will not see  it

I think that you should add a flag on the "sites" class that controls the availability of the drive module and set it only for civil